docs/tutorials/social-network-application/new-kuneiform-concepts.mdx

```diff
@@ -11,15 +11,14 @@ In this tutorial, we will expand on the functionality introduced in the [Hello W
 ## Prerequisites
```

Comment:
@brennanjl should i add action privacy information (e.g. owner, view, must_sign, etc) as a part of this tutorial?

Reply:
At least view.  Probably don't need the others

Reply:
addressed here: [0d326c6](https://github.com/kwilteam/docs/pull/34/commits/0d326c6371b4dff3b82bcbcc1e41dafa1900bc24)